src/components/datepicker/datepicker.spec.ts

```diff
@@ -1,92 +1,41 @@
+import ModulPlugin from '../../utils/modul/modul';
 import uuid from '../../utils/uuid/uuid';
 import DatepickerPlugin, { MDatepicker } from './datepicker';
-import ModulPlugin from '../../utils/modul/modul';
 
 
 
 jest.mock('../../utils/uuid/uuid');
 (uuid.generate as jest.Mock).mockReturnValue('uuid');
 
+
+
 describe('MDatepicker', () => {
     let localVue: VueConstructor<Vue>;
-    let mockDropDown: VueConstructor<Vue>;
+    let wrapper: any;
 
     beforeEach(() => {
         resetModulPlugins();
         localVue = createLocalVue();
         Vue.use(DatepickerPlugin);
         Vue.use(ModulPlugin);
-    });
-
-    it('should render correctly', () => {
-        const tp: Wrapper<MDatepicker> = mount(MDatepicker, {
-            localVue: localVue
-        });
-
-        return expect(renderComponent(tp.vm)).resolves.toMatchSnapshot();
-    });
-
-    it('should render correctly when waiting is set', () => {
-        const tp: Wrapper<MDatepicker> = mount(MDatepicker, {
-            localVue: localVue,
-            propsData: {
-                waiting: true
-            }
-        });
-
-        return expect(renderComponent(tp.vm)).resolves.toMatchSnapshot();
-    });
-
-    it('should render correctly when disabled is set', () => {
-        const tp: Wrapper<MDatepicker> = mount(MDatepicker, {
+        wrapper = mount(MDatepicker, {
+            sync: false,
             localVue: localVue,
-            propsData: {
-                disabled: true
+            stubs: {
+                transition: TransitionStub as any,
+                portal: PortalStub as any
             }
         });
-
-        return expect(renderComponent(tp.vm)).resolves.toMatchSnapshot();
-    });
-
-    it('should render correctly when error is set', () => {
-        const tp: Wrapper<MDatepicker> = mount(MDatepicker, {
-            localVue: localVue,
-            propsData: {
-                error: true,
-                errorMessage: 'Nostrud laboris quis velit voluptate aute elit elit non.'
-            }
-        });
-
-        return expect(renderComponent(tp.vm)).resolves.toMatchSnapshot();
-    });
-
-    it('should render correctly when valid is set', () => {
-        const tp: Wrapper<MDatepicker> = mount(MDatepicker, {
-            localVue: localVue,
-            propsData: {
-                valid: true,
-                validMessage: 'Nostrud laboris quis velit voluptate aute elit elit non.'
-            }
-        });
-
-        return expect(renderComponent(tp.vm)).resolves.toMatchSnapshot();
     });
 
-    it('should render correctly min/max time', () => {
-        const tp: Wrapper<MDatepicker> = mount(MDatepicker, {
-            localVue: localVue,
-            propsData: {
-                duration: true,
-                min: moment().subtract(2, 'years'),
-                max: moment().add(2, 'years')
-            }
-        });
+    it('When the field is click then popup show open and field has focus', async () => {
```

Comment:
is clicked then the popup appears and field has focus?

Reply:
jai fixer le test

Reply:
`When the field is click then popup show open and field has focus`
=>
`When the field is clicked then the popup is open and the field has focus`